faq/raycasting.md

```diff
@@ -21,17 +21,15 @@ Raycasting is a method of "drawing" a line from the head of a mob to detect wher
 > 
 > ```
 #> foo:start_raycast
-execute as @a anchored eyes positioned ^ ^ ^ run function foo:raycast
+execute as @a at @s anchored eyes positioned ^ ^ ^ run function foo:raycast
```

Comment:
Correct me if I'm wrong, but doesn't `anchored eyes` with local coordinates use the eyes, ignoring the position?

Reply:
Yes, but shouldn't the raycast start at the player?

Reply:
Oh, you're right.